jewelbots_nRF51822_board_v100/cores/JWB_nRF51822/Timer.cpp

```diff
@@ -56,5 +61,10 @@ extern "C"{
     }
   }
 
+  uint32_t Timer::millis(void) {
+    uint32_t num_ticks;
+    app_timer_cnt_get(&num_ticks); // TODO What to do with returned status?
```

Comment:
In this particular case, there is no reason to call APP_ERROR_CHECK because the app_timer_cnt_get function is hard coded to return NRF_SUCCESS no matter what it does (if a timer is not running it will return a zero, but not an error).